### ISSUE TITLE
Installing and setting up issues

### ISSUE THREAD

**Issue by JCengineer** [realXtend/WebTundra#87]
Hi,

I've been attempting to get the WebTundra app working locally. I have hit a number of challenges where I need help solving.
Grunt jsdoc isn't compiling correctly.
![image](https://cloud.githubusercontent.com/assets/3214978/12760130/ce27b3d2-c9dc-11e5-834f-209eff2ae3d3.png)
WebTundraEditor.webtundrajs isn't in the repository
![image](https://cloud.githubusercontent.com/assets/3214978/12760149/e074aa36-c9dc-11e5-968e-49370d6e230e.png)
Here's a screenshot of the files in the src directory.

Please advise.
James

**Comment by jonnenauha** [comment]
Yes this is a duplicate to https://github.com/realXtend/WebTundra/issues/86

I think the latest release and head has this bug.

Quick fix: comment out this line https://github.com/realXtend/WebTundra/blob/master/Gruntfile.js#L754 and run grunt build again.

**Comment by jonnenauha** [comment]
I've fixed the build with the quick fix. Lets track the proper fix in #86.

https://github.com/realXtend/WebTundra/commit/5171d06155876a013748c43d21e7ad62625a0ab3

Pull latest head and run `grunt build` again, skip the `grunt doc` part for now.

@cvetan5 Can we make a hotfix release to FIWARE to fix this properly? Maybe we do that later when doc generation has also been fixed.

**Comment by jonnenauha** [comment]
Btw. `grunt dev` is a blocking command that will start a web server and host the dev environment (should auto open a tab to the dev client). You can kill it with `Ctrl+c`.

**Comment by JCengineer** [comment]
Thanks.

I spotted the 'grunt dev' server running this morning. Usually the server logs 'Listening on port XXX' so I was blind to the fact that it was running! Thanks for the help.

I've having trouble with the example scenes too. Do you want me to start new issues?
I'm running TundraConsole.exe --server --headless --d3d9 --nullrenderer --file "<file_name_below>" and accessing it through "http://localhost:8082/html/client.html" on another locally networked computer.
"scenes/AssImpDemo/scene.txml"
![image](https://cloud.githubusercontent.com/assets/3214978/12786761/5a610e90-ca8a-11e5-9f27-e20a9e7d1573.png)
"scenes/DayNight/scene.txml"
![image](https://cloud.githubusercontent.com/assets/3214978/12786796/7f426b14-ca8a-11e5-8b20-76f0d09704f2.png)
